testing/integration/test/bin/load-zork.sh

```diff
@@ -47,14 +51,20 @@ if $RUNVNC; then
     x11vnc -display :10 -forever &
 fi
 
-if ! $PREBUILT; then
+if [[! $PREBUILT ]] || $NPM; then
     mkdir -p /test/src
     cd /test/src
     npm install -g bower grunt-cli
-    echo git clone $CLONEARGS $CLONESRC
-    git clone $CLONEARGS $CLONESRC
-    cd uproxy-lib
-    ./setup.sh install
+    if $NPM; then
+        npm install uproxy-lib
+        cp -r node_modules/uproxy-lib .
```

Comment:
Copying the whole folder is a little wasteful. The important thing is that the extension is made available at:
`/test/src/uproxy-lib/build/dev/uproxy-lib/samples/zork-{chrome|firefox}app`

I don't think there's any way to make NPM install the package directly there without a `node_modules` folder but you should be able to do something with `npm install --prefix` and an `ln -s`, e.g.:
```bash
npm install --prefix /test/src uproxy-lib
ln -s /test/src/node_modules /test/src/uproxy-lib
```